pom.xml

```diff
@@ -1329,7 +1329,7 @@
                   <version>${java.version}</version>
                 </requireJavaVersion>
                 <requireMavenVersion>
-                  <version>3.3.9</version>
+                  <version>3.8.6</version>
```

Comment:
I updated issue title to make it more clear
https://github.com/checkstyle/checkstyle/issues/13970